packages/jamulus/PKGBUILD

```diff
@@ -0,0 +1,50 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Jose Riha <jose1711 gmail com>
+# Contributor: Julien Taverna <jujudusud gmail com>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+pkgbase=jamulus
+pkgname=(jamulus jamulus-server)
+pkgver=3.8.2
+pkgrel=2
+pkgdesc="Internet jam session software"
+arch=(x86_64 aarch64)
+url='https://jamulus.io/'
+license=(GPL2)
```

Comment:
c35ccc36617c75c73a4bea06a8954d42259c79c9

However I wasn't 100 % sure since for example the [faust package](https://archlinux.org/packages/community/x86_64/faust/) is only listed with GPL2 but also has MIT and STK-licensed parts in their source code (and standard library).

After all, MIT/BSD-style licenses are often GPL-compatible so in theory we could probably choose to only apply GPL to our package...